client/src/app/site/motions/services/motion-pdf-catalog.service.ts

```diff
@@ -53,10 +53,20 @@ export class MotionPdfCatalogService {
         const motionDocList = [];
         const printToc = exportInfo.pdfOptions.includes(`toc`);
         const enforcePageBreaks = exportInfo.pdfOptions.includes(`addBreaks`);
+        const contentFlowText = exportInfo.content.includes(`flowText`);
 
         for (let motionIndex = 0; motionIndex < motions.length; ++motionIndex) {
+            let flowText = contentFlowText;
             try {
-                const motionDocDef: any = this.motionPdfService.motionToDocDef(motions[motionIndex], exportInfo);
+                if (motionIndex === 0) {
+                    flowText = false;
+                }
+
+                const motionDocDef: any = this.motionPdfService.motionToDocDef(
+                    motions[motionIndex],
+                    exportInfo,
+                    flowText
+                );
```

Comment:
The variable `flowText` does not seem to be necessary: Just pass `contentFlowText || motionIndex !== 0`.